website/docs/d/client_config.html.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: "azurestack"
+page_title: "Azure Resource Manager: azurestack_client_config"
+sidebar_current: "docs-azurestack-datasource-client-config"
+description: |-
+  Get information about the configuration of the azurestack provider.
+---
+
+# Data Source: azurestack_client_config
+
+Use this data source to access the configuration of the Azure Resource Manager
+provider.
+
+## Example Usage
+
+```hcl
+data "azurestack_client_config" "current" {}
+
+output "account_id" {
+  value = "${data.azurestack_client_config.current.service_principal_application_id}"
+}
+```
+
+## Argument Reference
+
+There are no arguments available for this data source.
+
+## Attributes Reference
+
+* `client_id` is set to the Azure Client ID (Application Object ID).
+* `tenant_id` is set to the Azure Tenant ID.
+* `subscription_id` is set to the Azure Subscription ID.
+
+---
+
+~> **Note:** the following fields are only available when authenticating via a Service Principal (as opposed to using the Azure CLI):
```

Comment:
we don't support the Azure CLI for Azure Stack - as such this note can be removed?